File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L6/05/method.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function beautyReverse(num: number): void {
  for (; num > 0; num = Math.floor(num / 100 * 10)) {
    console.log(num % 10);
  }
  console.log();
}

function reverse(): void {
  rl.question('Enter an integer: ', (input) => {
    const n = parseInt(input);
    beautyReverse(n);
    rl.close();
  });
}

reverse();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L3/05/method.kt
package IR

import java.util.Scanner

fun beautyReverse(num: Int) {
    var tempNum = num
    while (tempNum != 0) {
        val r = tempNum % 10
        print(r)
        tempNum /= 10
    }
    println()
}

fun main(args: Array<String>) {
    print("Enter an integer: ")
    val input = Scanner(System.`in`)
    val n = input.nextInt()
    beautyReverse(n)
}

